src/colors.yaml

```diff
@@ -56,6 +59,10 @@ ui:
       dimmingOverlay: [0,0,0,0.4]
       tappable: [56,56,15,0.09]
       tappableInverted: [255,246,196,0.27]
+      fieldBG: [0,0,0,0.09]
```

Comment:
does it make sense to set these by color name in the base palette and map to feature-specific names in sasstools or MWC?